pages/custom_blocks.py

```diff
@@ -8,6 +8,41 @@
 from api.serializers import ImageSerializer
 from openstax.functions import build_image_url, build_document_url
 
+class HeroBlock(blocks.StructBlock):
+    heading = blocks.CharBlock(required=True)
+    sub_heading = blocks.CharBlock(required=False)
+    description = blocks.RichTextBlock(required=False)
+    image = ImageChooserBlock(required=False)
+    primary_cta_text = blocks.CharBlock(required=False)
+    primary_cta_link = blocks.URLBlock(required=False)
+    secondary_cta_text = blocks.CharBlock(required=False)
+    secondary_cta_link = blocks.URLBlock(required=False)
+
+    class Meta:
+        icon = 'image'
+
+class CardBlock(blocks.StructBlock):
```

Comment:
this seems to be `card block` + `card` in one thing. shouldn't it have `style` and then a list of cards separately?